cura/LayerDataBuilder.py

```diff
@@ -48,28 +48,63 @@ def setLayerThickness(self, layer, thickness):
 
         self._layers[layer].setThickness(thickness)
 
-    def build(self):
+    #   material color map: [r, g, b, a] for each extruder row.
+    #   line_type_brightness: compatibility layer view uses line type brightness of 0.5
+    def build(self, material_color_map, line_type_brightness = 1.0):
```

Comment:
Could you use doxygen style markup here? Makes it a bit more consistent (use \param & \type{dict} etc)